src/intervals/interval_operations/numeric.jl

```diff
@@ -127,39 +139,94 @@ Return the midpoint of an interval `a` together with its radius.
 Function required by the IEEE Standard 1788-2015 in Section 10.5.9 for the
 set-based flavor.
 """
-function midradius(a::Interval{T}) where {T<:NumTypes}
-    isempty_interval(a) && return convert(T, NaN), convert(T, NaN)
+function midradius(a::BareInterval{T}) where {T<:NumTypes}
     m = mid(a)
     return m, max(m - inf(a), sup(a) - m)
 end
 
-midradius(a::Real) = (mid(a), radius(a))
-
 """
     mag(a::Interval)
 
 Magnitude of an interval. Return `NaN` for empty intervals.
 
 Implement the `mag` function of the IEEE Standard 1788-2015 (Table 9.2).
 """
-function mag(a::Interval{T}) where {T<:NumTypes}
+function mag(a::BareInterval{T}) where {T<:NumTypes}
     isempty_interval(a) && return convert(T, NaN)
     return max(abs(inf(a)), abs(sup(a)))
 end
 
-mag(a::Real) = abs(a)
-
 """
     mig(a::Interval)
 
 Mignitude of an interval. Return `NaN` for empty intervals.
 
 Implement the `mig` function of the IEEE Standard 1788-2015 (Table 9.2).
 """
-function mig(a::Interval{T}) where {T<:NumTypes}
+function mig(a::BareInterval{T}) where {T<:NumTypes}
     isempty_interval(a) && return convert(T, NaN)
     in_interval(0, a) && return zero(T)
     return min(abs(inf(a)), abs(sup(a)))
 end
 
-mig(a::Real) = abs(a)
+dist(a::BareInterval, b::BareInterval) = max(abs(inf(a)-inf(b)), abs(sup(a)-sup(b)))
```

Comment:
Docstrings?

Reply:
Not sure where this function came from, I do not see it in the standard (maybe I missed it 🤷‍♂️). But I added a small docstring regardlesss.

Reply:
I think it comes from the book of Tucker, though I should need to have a look. If I recall correctly, it is the Housdorff distance